src/settings-ui/Settings.UI/SettingsXAML/Controls/ShortcutControl/ShortcutDialogContentControl.xaml

```diff
@@ -21,7 +24,7 @@
             x:Name="KeysControl"
             Grid.Row="1"
             Height="56"
-            Margin="0,64,0,0"
+            Margin="0,40,0,0"
```

Comment:
Something changed here, now when the `InfoBar` is shown the dialog "jumps" which isn't a great experience.

New vs. old
![Jumpingshortcutdialog](https://github.com/microsoft/PowerToys/assets/9866362/acb3c3d0-95d4-4440-b970-e4d0f4638911)

Reply:
Ahh yes, I was wondering why the height was hard-coded... Maybe insert a grid _row_ for the InfoBar(s) with a MinHeight?

Reply:
Double check what there was originally and revert any changes there? (maybe it was the margin decrease?) - just to make sure we don't run into any other issues.

Reply:
The row actually works well 🙂 check out the update?

Reply:
Works! But now the dialog height has increased? 

![image](https://github.com/microsoft/PowerToys/assets/9866362/2a9fba5f-7d6e-4a08-b6f2-d20d8e1b3c7d)

Reply:
I'll tweak it

Reply:
![image](https://github.com/microsoft/PowerToys/assets/65828559/db4a8bf3-ea90-4e37-9bdf-981a4417f1e6)
↳ Note: removing the **MinWidth** solves issue with text being cut off. It happens because the minimum with of the parent window and the ShortcutDialogContentControl were both **480**. Is it bad if the gutters are not exactly what they were?
The formatting of the tooltip text is off-topic, but good to know.

Reply:
Off-topic: looks like the majority of MS app and Settings use a minwidth of 500.

Reply:
Oh yeah we can increase the MinWidth is needed.. it's up to the content of the dialog to see chose what value makes sense there. We just need to make sure that upon resizing things don't get clipped.

Dialog height increase is fine I guess.. the spacing between the infobar, the shortcut and text feels a bit off though?